app/src/organisms/OnDeviceDisplay/Navigation/NavigationMenu.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import { useDispatch } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  Icon,
+  JUSTIFY_CENTER,
+  TYPOGRAPHY,
+  SPACING,
+} from '@opentrons/components'
+
+import { StyledText } from '../../../atoms/text'
+import { ModalShell } from '../../../molecules/Modal'
+import { home, ROBOT } from '../../../redux/robot-controls'
+import { restartRobot } from '../../../redux/robot-admin'
+import { useLights } from '../../Devices/hooks'
+
+import type { Dispatch } from '../../../redux/types'
+
+interface NavigationMenuProps {
+  onClick: React.MouseEventHandler
+  robotName: string
+}
+
+export function NavigationMenu(props: NavigationMenuProps): JSX.Element {
+  const { onClick, robotName } = props
+  const { t, i18n } = useTranslation(['devices_landing', 'robot_controls'])
+  const { toggleLights } = useLights()
+  const dispatch = useDispatch<Dispatch>()
+
+  return (
+    <ModalShell
+      borderRadius={BORDERS.size_three}
+      onOutsideClick={onClick}
+      width="18.0625rem"
+      isOnDeviceDisplay
+    >
+      <Flex
+        flexDirection={DIRECTION_COLUMN}
+        justifyContent={JUSTIFY_CENTER}
+        padding={`0rem ${SPACING.spacingL}`}
+      >
+        <Flex
+          alignItems={ALIGN_CENTER}
+          height="4.875rem"
+          onClick={() => dispatch(home(robotName, ROBOT))}
+        >
+          <Icon
+            name="reset-position"
+            aria-label="reset-position_icon"
+            size="2rem"
```

Comment:
```suggestion
            size={SIZE_2}
```

maybe we should update size constants 🤔